README.md

```diff
@@ -1,11 +1,101 @@
 
-Corporate Staff Roster data api service.
+## Continuous Integration  
+https://app.circleci.com/pipelines/github/ministryofjustice/csr-api
 
-## Service Description
+## Running locally  
+  
+### Prerequisites  
+* Java JDK 11+  
+* An editor/IDE
+* Gradle  
+* Docker  
+* OAuth  [(running in a container)](#oauth-security)
+  
+#### OAuth security  
+In order to run the service locally, [Nomis OAuth Service](https://github.com/ministryofjustice/nomis-oauth2-server/) is required. This can be run locally using the [docker-compose.yml](docker-compose.yml) file which will pull down the latest version.  From the command line run:
+  
+```
+ docker-compose up 
+```  
 
-## Setup and Run
+### Build service and run tests  
+This service is built using Gradle. In order to build the project from the command line and run the tests, use:
+```  
+./gradlew clean build  
+```  
+The created JAR file will be named "`csr-api-<yyyy-mm-dd>.jar`", using the date that the build takes place in the format `yyyy-mm-dd`. 
 
-## Terminology
+### Start the application with H2 database  
+The configuration can be changed for the api to use an in-memory H2 database by using the spring boot profile `dev`. On the command line run:
+  ```  
+SPRING_PROFILES_ACTIVE=dev 
+java -jar build/libs/csr-api-<yyyy-mm-dd>.jar  
+```  
 
-## Sql
+### Start the application with Oracle database  
+This configuration can be changed to use a VM Oracle database using the spring boot profile `oracle`.  On the command line run:
+  ```  
+SPRING_PROFILES_ACTIVE=oracle 
+java -jar build/libs/csr-api-<yyyy-mm-dd>.jar  
+```  
+Flyway will not run and you will also need to provide values for each schema (1-6) like so:
+
+ ```
+ CSR_REGION1_URL
+ CSR_REGION1_USERNAME
+ CSR_REGION1_PASSWORD
+ CSR_REGION1_SCHEMA
+```
+
+### Additional configuration  
+The application is configurable with conventional Spring parameters.  
+The Spring documentation can be found here: https://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html  
+  
+#### Default port  
+By default the application starts on port '8081'.   To override, set server.port (e.g. `SERVER_PORT=8099 java -jar build/libs/csr-api-<yyyy-mm-dd>.jar` )  
+  
+### Documentation  
+The generated documentation for the api can be viewed at http://localhost:8081/swagger-ui.html  
+  
+### Health  
+  
+- `/ping`: will respond `pong` to all requests.  This should be used by dependent systems to check connectivity to   
+offender assessment service, rather than calling the `/health` endpoint.  
+- `/health`: provides information about the application health and its dependencies.  This should only be used  
+by offender assessment service health monitoring (e.g. pager duty) and not other systems who wish to find out the   
```

Comment:
I was going to complement you on this README being a work of art but you forgot to cover up your copy pasting 😉

Reply:
you'll be the first and last person to read it.